peerconnection.go

```diff
@@ -391,6 +396,11 @@ func (pc *PeerConnection) GetConfiguration() Configuration {
 	return pc.configuration
 }
 
+// ObjectID returns the connection object id
+func (pc *PeerConnection) ObjectID() string {
```

Comment:
The W3C has pushed back pretty hard on having ids assigned to anything https://www.w3.org/2018/04/12-webrtc-minutes#item06

I would avoid if possible, we are going to break WASM/Go interop if we do go down that route.